Doc/library/sqlite3.rst

```diff
@@ -417,6 +417,17 @@ Module constants
    This flag may be combined with :const:`PARSE_COLNAMES` using the ``|``
    (bitwise or) operator.
 
+.. data:: SQLITE_OK
+          SQLITE_DENY
+          SQLITE_IGNORE
+
+   Flags that should be returned by the *authorizer_callback* callable
+   passed to :meth:`Connection.set_authorizer`, to indicate whether:
+
+   * Access is allowed (:const:`!SQLITE_OK`),
```

Comment:
Oh, I forgot the accidental comma! Note to self to fix it in an upcoming PR.

Reply:
Silly mistake; my fault again for being sloppy here

Reply:
No problem, you're fine!